### ISSUE TITLE
[Bug Report] This does not seem to work for UK

### ISSUE THREAD

**Issue by MithUK1** [ollo69/ha-smartthinq-sensors#59]
Hi, when i select my area as UK the URL i am given still seems to be for the US, therefore It doesn;t know my credentials and I cant get a response URL to input.

your help with this would be greatly appreciated

**Comment by ollo69** [comment]
Please provide more detail about parameters used by you for component configuration or I will not be able to help you.

**Comment by KTibow** [comment]
Mind showing some screenshots of setup process @MithUK1?

**Comment by MithUK1** [comment]
Hi, sorry for the delay - these are the steps i take, installed on HASS via HACS:
![step1](https://user-images.githubusercontent.com/60322560/86836023-d0dc1b00-c094-11ea-8fb9-3d2cf0bb7f14.jpg)
![step2](https://user-images.githubusercontent.com/60322560/86836030-d2a5de80-c094-11ea-9d56-db9b615418f5.jpg)
![step3](https://user-images.githubusercontent.com/60322560/86836035-d46fa200-c094-11ea-8b39-b684b4a9c7c2.jpg)
![step4](https://user-images.githubusercontent.com/60322560/86836041-d6396580-c094-11ea-9e3c-d3f5e4770bfd.jpg)

**Comment by MithUK1** [comment]
closed by accident -= reopened

**Comment by ollo69** [comment]
I think you should use:
Region: GB
Languange: en-GB

**Comment by ollo69** [comment]
I released a new version that better clarify allowed code.
With new release (v0.5.5) you should use:

Region: GB
Language: en

**Comment by ollo69** [comment]
@MithUK1,

can please provide feed-back and close this issue if solved.

**Comment by bmwm69** [comment]
Hi @ollo69 

I have similar problem, I have standalone account and works good with sampsyo fork , but i would like test your fork 


Config Settings:

Region: PT
Language: pt

after login redirect to https://kr.m.lgaccount.com/login/iabClose?state=xxxxx and none device appeared in Integration its possible view any log ?

**Comment by KTibow** [comment]
@bmwm69 did you read the instructions? You need to copy the URL back into Home Assistant in order to set it up.

**Comment by bmwm69** [comment]
@KTibow  Yes , I copy the URL back

Some images to help understand :|

![image](https://user-images.githubusercontent.com/33760373/87552564-c24fbe00-c6a9-11ea-934f-b7bb92406f91.png)

![image](https://user-images.githubusercontent.com/33760373/87552598-ce3b8000-c6a9-11ea-9f18-3424515d7be9.png)

After login:

![image](https://user-images.githubusercontent.com/33760373/87552668-e57a6d80-c6a9-11ea-9014-1db7a97d96a5.png)
![image](https://user-images.githubusercontent.com/33760373/87552737-faef9780-c6a9-11ea-97b3-2170ac1cf69a.png)

I already tried change kr to pt but same problem :\

**Comment by KTibow** [comment]
Can you view your devices in the app? You might have signed up with a social account, that might cause problems.

**Comment by bmwm69** [comment]
Yes I can view devices in the app , no its a LG Account

**Comment by bmwm69** [comment]
Create a New LG account with location GB and still without any device :/

**Comment by ollo69** [comment]
Just to clarify, is not enough create a new account, you must use the same account in the LG phone app, registering your devices with this account, and than in the integration.
Are you sure you perform all this step?

**Comment by bmwm69** [comment]
Yes , created new LG account on my phone and add one device , after that I tried add that account into integration. and still without any device.

Is possible check any log ?

**Comment by KTibow** [comment]
@bmwm69 logs: `/config/logs`

**Comment by ollo69** [comment]
Sorry but I realize just now that you have only climate device that are not yet supported by this integration

**Comment by modem** [comment]
> 
> 
> Hi @ollo69
> 
> I have similar problem, I have standalone account and works good with sampsyo fork , but i would like test your fork
> 
> Config Settings:
> 
> Region: PT
> Language: pt
> 
> after login redirect to https://kr.m.lgaccount.com/login/iabClose?state=xxxxx and none device appeared in Integration its possible view any log ?
> 
> I only have 3 air conditioners

Hi, can you share details on how you integrated your AC with the sampsyo fork (did you use this one https://github.com/sampsyo/hass-smartthinq)?
I'm stuck with a wideq token error: wideq.core.TokenError
